docs/integrations/saas-cloud/asana.md

```diff
@@ -69,10 +69,13 @@ To set up [Cloud to Cloud Integration Asana Source](/docs/send-data/hosted-colle
 
 ## Installing the Asana app
 
-This section describes how to install the Asana app for Sumo Logic and description of the dashboard. You'll also need to configure the [Asana Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source/).
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Asana dashboard
```

Comment:
Although there's only one Asana dashboard, I'd still make this plural (_dashboards_) for consistency with its counterpart docs. We may add more in the future.

```suggestion
## Viewing Asana dashboards
```